tools/prune_images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Description: This script can be used to prune the images stored in the /docs/source/images directory.
+#              It will delete any images which are not referenced anywhere else in the Mantid repo.
+#
+# Usage: ./prune_images.sh --dry-run
```

Comment:
the script will only work from the root of the repo, so it might be better to change the usage instructions
```suggestion
# Usage: ./tools/prune_images.sh --dry-run
```